stdlib/bir/src/main/ballerina/bir/bir_emitter.bal

```diff
@@ -144,6 +152,10 @@ public type BirEmitter object {
         print("\t|\t");
         self.opEmitter.emitOp(errorEntry.errorOp);
     }
+
+    function emitVis(int i) {
+
```

Comment:
Do we need this function. Cannot see any usages.

Reply:
yes, seems not used. shall we remove